client/shared/src/test/scala/org/http4s/client/ClientRouteTestBattery.scala

```diff
@@ -129,6 +129,51 @@ abstract class ClientRouteTestBattery(name: String) extends Http4sSuite with Htt
     }
   }
 
+  GetRoutes.getPaths.get(GetRoutes.SimplePath).foreach { expected =>
+    val path = GetRoutes.SimplePath
+
+    test(s"$name Execute GET withEntity $path") {
+      serverClient().flatMap { case (server, client) =>
+        val address = server().addresses.head
+        val req =
+          Request[IO](uri = Uri.fromString(s"http://$address$path").yolo).withEntity("entity")
+        client()
+          .run(req)
+          .use(resp => expected.flatMap(checkResponse(resp, _)))
+          .assert
+      }
+    }
+
+    test(s"$name Execute GET withEntity then withHeaders $path") {
```

Comment:
Note: this test confuses ember if run without `normalize` (say if it were added to #6111 ). 
The test itself succeeds but another (the next?) test in the suite starts failing.

Just thought to mention it because it may indicate an issue in ember.

Reply:
Yes, thanks for mentioning this. It reminds me of https://github.com/http4s/http4s/issues/5256, which also had the symptom of the _next_ test failing.